<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wReleaseNotes</class>
 <widget class="QMainWindow" name="wReleaseNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">wReleaseNotes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressources.qrc">
    <normaloff>:/rec/data/icons/iconSwitch.svg</normaloff>:/rec/data/icons/iconSwitch.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
	 <widget class="QLabel" name="lblNewUpdate">
	  <property name="text">
	   <string>Release notes:</string>
	  </property>
	  <property name="alignment">
	   <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
	  </property>
	 </widget>
	</item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
	 <enum>Qt::Horizontal</enum>
	</property>
	<property name="sizeHint" stdset="0">
	 <size>
	  <width>40</width>
	  <height>20</height>
	 </size>
	</property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
	 <widget class="QLabel" name="lblCurrentVersion">
	  <property name="text">
	   <string notr="true">currentVersion</string>
	  </property>
	 </widget>
	</item>
	<item>
	 <widget class="QLabel" name="lblNewVersion">
	  <property name="text">
	   <string notr="true">newVersion</string>
	  </property>
	 </widget>
	</item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="teedReleaseNotes">
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbxClearAppDir">
      <property name="text">
       <string>Clear application directory</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblClearAppDirInfo">
      <property name="styleSheet">
       <string notr="true">color: red</string>
      </property>
      <property name="text">
       <string>Attention: This option deletes the entire application directory including all files (even those that do not originate from the application).</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
	 <enum>Qt::Horizontal</enum>
	</property>
	<property name="sizeHint" stdset="0">
	 <size>
	  <width>40</width>
	  <height>20</height>
	 </size>
	</property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnUpdateNow">
        <property name="font">
	 <font>
	  <bold>true</bold>
	 </font>
	</property>
	<property name="text">
	 <string>Download and install</string>
	</property>
	<property name="icon">
	 <iconset resource="../../ressources.qrc">
	  <normaloff>:/rec/data/icons/iconDownload.svg</normaloff>:/rec/data/icons/iconDownload.svg</iconset>
	</property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
	 <string>Close</string>
	</property>
	<property name="icon">
	 <iconset resource="../../ressources.qrc">
	  <normaloff>:/rec/data/icons/iconQuit.svg</normaloff>:/rec/data/icons/iconQuit.svg</iconset>
	</property>
	<property name="shortcut">
	 <string notr="true">Esc</string>
	</property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
